core/main/src/broker/endpoint_broker.rs

```diff
@@ -326,6 +326,16 @@ impl EndpointBrokerState {
         rule: Rule,
         extn_message: Option<ExtnMessage>,
     ) -> BrokerRequest {
+        let (_id, req) = self.update_request2(rpc_request, rule, extn_message);
+        req
+    }
+
+    fn update_request2(
```

Comment:
This is virtually identical to `update_request()` [here](https://github.com/rdkcentral/Ripple/blob/7f3ebb7d4bd2708499f8b93da7bddf99a3518861/core/main/src/broker/endpoint_broker.rs#L323), can we just modify the existing method as necessary?

Reply:
Fixed